bundle-workflow/src/test_workflow/docs/README.md

```diff
@@ -0,0 +1,125 @@
+# OpenSearch CI CD Test Infrastructure
```

Comment:
Should not be under `test_workflow/docs/README`, but just `test_workflow/README`.

Reply:
Hey @dblock , the reason was to have docs and related images under one folder `docs/` instead of having alongside code files, but i'm ok with the change you suggested. Where should we put png image that is referenced inside this readme?

Reply:
An img subfolder is cool with me.

Reply:
An img subfolder is cool with me.